ormconfig.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { config } = require('dotenv');
+
+config();
+
+const dbConfig = {
+  host: process.env.DB_URL,
+  port: process.env.DB_PORT || (process.env.DB_LOCAL !== '' ? 5432 : 54320),
```

Comment:
If `is_setup_docker` is true use 54320 else 5432.

Reply:
Yes, but I'd also keep the DB_PORT because I could have my local db on a different port or use a remote db. For example hosting it on a free online db as a service

Reply:
@Zeko369 Yes sure we should keep db_port.